portfolio/src/main/webapp/gallery.html

```diff
@@ -1,4 +1,4 @@
-``<!DOCTYPE html>
+<!DOCTYPE html>
 <html>
 
 <head>
```

Comment:
Also indent the <head> section of this file

Reply:
Noted.